continuous-integration/use-cases/markmap.md

```diff
@@ -0,0 +1,65 @@
+* Continuous Integration
+    * For Analysis and Testing
+        * For verification, notification and assembly
+            * [Git hooks](https://git-scm.com/docs/githooks)
```

Comment:
In the documentation guide, we used the prefix "Using" or "Implementing", etc., depending on the nature of the tool or recommendation. Advise using a proper prefix depending on the nature of the recommendation. e.g. if it is a tool, use "Using", or if it is a template, use "Implementing".